ui/webui/src/components/installation/InstallationProgress.jsx

```diff
@@ -69,6 +87,7 @@ export class InstallationProgress extends React.Component {
                         });
                         taskProxy.addEventListener("Succeeded", () => {
                             this.setState({ status: "success" });
```

Comment:
You can do:
`this.setState({ status: "success", currentProgressStep: 4 });`

Reply:
remember every time you call setState the component 're-renders'. React is smart enough to group setStates that are directly after each each, but if you just group them in the code as well, it's a good practice.